core/components/organisms/combobox/trigger/MultiselectTrigger.tsx

```diff
@@ -119,7 +119,10 @@ export const MultiSelectTrigger = (props: MultiSelectTriggerProps) => {
     ...rest
   } = props;
 
-  const inputRef = React.createRef<HTMLInputElement>();
+  const localInputRef = React.useRef<HTMLInputElement>();
+  const customRef = React.useRef<any>();
+  const inputRefToUse = (forwardedInputRef || localInputRef) as React.RefObject<HTMLInputElement>;
```

Comment:
Please rename this variables to be more appropriate names according to their role